### ISSUE TITLE
Consider making the user's editor configuration machine-specific rather than global

### ISSUE THREAD

**Issue by fahhem** [Reviewable/Reviewable#1108]
I use Reviewable on my laptop and desktop, but my folder structure is not the same between them.

For one, my laptop runs Windows so the path to my project on disk is under `C:\Users\...`, and my desktop runs Linux and the path is under `/home/...`. Secondly, I organize my folders differently on the two computers, so even if I had something like `$HOME` or `%HOMEPATH%` (what Windows calls `$HOME`), and the slash/backslash controversy were handled, they would still be in different folders with different names.

I think it makes sense that editor configurations are tied to the machine. We could continue to store the first editor config in the user profile, but if a user edits the config on a machine, it should only override it for that one, and leave the original value in place for any other machine they use.

**Comment by pkaminski** [comment]
Makes sense, though I think we'd need to be a bit more clever than just "first setting becomes global default (and can only be changed from the original machine)".  Perhaps each machine keeps a local setting, and the last one set also sets the global default for any machines without a local override.  That should be pretty easy to implement and follows the general pattern of default/specific settings in Reviewable.

But I'd still recommend normalizing your path structure across machines, for your own sanity if nothing else!  :D
